src/site/content/en/blog/better-youtube-web-part1/index.md

```diff
@@ -0,0 +1,180 @@
+users to control features such as playback speed and progress, had become
+over-componentized over time. 
+
+
+<figure>
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/qh9N7ZdkxylN1vYOOdUO.png", alt="YouTube player and controls visualized", width="800", height="300" %}
+  <figcaption>
+    The YouTube video player allows users to control playback speed, follow progress, skip sections, and others. When a user taps on a particular control, the change in the state must be communicated to other controls e.g. a user tap on the progress bar must be shared to controls such as play-head, subtitles etc.
+  </figcaption>
+</figure>
+
+Each progress bar touch-move event triggered two
+extra style recalculations and took 21.17 ms during performance test runs in the lab. As new controls were added over time, the pattern of decentralized control would often cause circular dependencies and memory leaks, negatively impacting watch page performance.
+
+<figure>
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/vCTMilflEoDnkw1p8EuU.png", alt="21.17ms event shown on the Performance timeline", width="800", height="202" %}
+  <figcaption>
+    Chrome DevTools with a 4x CPU slow-down performance run. 
+  </figcaption>
+</figure>
+
+To fix the issues due to decentralized control, the team updated the player UI
+to synchronize all updates, essentially refactoring the player to one top-level
+component that would pass down data to its children. This ensured only one UI
+update (render) cycle for any state change, eliminating chained updates. The new
+player progress bar touch-move event has no style recalculations during its
+JavaScript execution and now only requires 25% the time of the old player.
+
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/E1EbOYuT2q7mDqfmR1zz.png", alt="Reduced time in events shown on the performance timeline", width="800", height="90" %}
+
+This code modernization also resulted in additional performance improvements
+such as improved watch load times on old devices, fewer abandoned playbacks, and
+a reduced number of non-fatal errors.
+
+## Conclusion
+
+As a result of YouTube's investment in performance, watch pages load much faster with 76% of YouTube's mobile website URLs now passing Core Web Vitals metric thresholds in the field. On desktop, lab LCP for the watch page was reduced from approximately 4.6 seconds to 1.6 seconds. Interaction and rendering performance of the site, especially on the YouTube video player, are seeing up to 75% less time spent in JavaScript execution than before.
+
+{% Aside 'success' %}
+76% of YouTube's mobile web URLs now pass the Core Web Vitals metrics, with improvements to business metrics like watch time as a result.
+{% endAside %}
+
+Improvements to the performance of YouTube web over the last year have also improved business metrics, including watch time and daily active users. We look forward to exploring how more of these performance optimizations can be applied to the YouTube desktop site.
```

Comment:
I think this last sentence can be dropped (or reworded to talk about general future optimizations) now that the post doesn't focus on the specific mobile and desktop changes. Not sure if this is true, but maybe something inspirational like:

"Based on the success of these efforts, we plan to continue exploring even more ways to optimize in the future."

Reply:
Sounds good! I'll go ahead and drop this last line and replace with your suggested wording.